src/admin-views/create-organizer-fields.php

```diff
@@ -1,21 +1,22 @@
 <tr class="linked-post organizer">
-	<td><?php esc_html_e( 'Phone:', 'the-events-calendar' ); ?></td>
+	<td><label for="organizer-phone"><?php esc_html_e( 'Phone:', 'the-events-calendar' ); ?></label></td>
 	<td>
-		<input tabindex="<?php tribe_events_tab_index(); ?>" type='text' name='organizer[Phone][]' class='organizer-phone' size='25' value='' />
+		<input id="organizer-phone" type='text' name='organizer[Phone][]' class='organizer-phone' size='25' value='' />
 	</td>
 </tr>
 <tr class="linked-post organizer">
-	<td><?php esc_html_e( 'Website:', 'the-events-calendar' ); ?></td>
+    <td><label for="organizer-website"><?php esc_html_e( 'Website:', 'the-events-calendar' ); ?></label></td>
 	<td>
-		<input tabindex="<?php tribe_events_tab_index(); ?>" type='text' name='organizer[Website][]' class='organizer-website' size='25' value='' />
+		<input id="organizer-website" type='text' name='organizer[Website][]' class='organizer-website' size='25' value='' />
 	</td>
 </tr>
 <tr class="linked-post organizer">
 	<td>
-		<?php esc_html_e( 'Email:', 'the-events-calendar' ); ?>
-		<small><?php esc_html_e( 'The e-mail address will be obfuscated on your site to avoid it getting harvested by spammers.', 'the-events-calendar' ); ?></small>
+        <label for="organizer-email"><?php esc_html_e( 'Email:', 'the-events-calendar' ); ?></label>
 	</td>
 	<td class="organizer-email">
-		<input tabindex="<?php tribe_events_tab_index(); ?>" type='text' name='organizer[Email][]' class='organizer-email' size='25' value='' />
+		<input id="organizer-email" type='text' name='organizer[Email][]' class='organizer-email' size='25' value='' />
+        <small><?php esc_html_e( 'The e-mail address will be obfuscated on your site to avoid it getting harvested by spammers.', 'the-events-calendar' ); ?></small>
```

Comment:
Need to fix the initial indent here also (it's currently using spaces rather than tabs, so looks odd anytime tabs aren't represented with exactly 4 chars).